src/components/route/route.js

```diff
@@ -41,6 +49,9 @@ const Route = ({ title, component: Component, dataRequired }) => {
     }
   }, [dataRequired, lroStatus, navigate]);
 
+  const displayTitle = (isPlacesPreview && overrides.isPlacesPreview.title) || title;
```

Comment:
ternary operator fits better here

Reply:
I don't necessarily agree. In case of the flag being true it should return override title (if exists) or normal title as fallback.
I any other case it should just return title. The tertiary counterpart would be:
`isPlacesPreview ? overrides.isPlacesPreview.title || title : title` or 
`isPlacesPreview && overrides.isPlacesPreview.title ?  overrides.isPlacesPreview.title : title`
For me personally, this looks uglier

Reply:
uglier cause its longer? shorter is not equal better.
Here I actually meant isPlacesPreview ? overrides.isPlacesPreview.title : title
I did not see at a glance that important logic here is also to fallback to title when isPlacesPreview.title is falsy.
Unintended usage of operators (&& in this case) may make the code shorter, but it doesn't make it easier to read